setup.cfg

```diff
@@ -0,0 +1,3 @@
+[pep8]
+exclude=lib
```

Comment:
Lets also exclude protobuf generated files(`sdk/artifacts/proto`)

Reply:
Maybe we should also exclude `examples`? Because they are mostly downloaded from tutorials or open source projects and would be better to keep them in the original style.

Reply:
Maybe we should also exclude examples? Because they are mostly downloaded from tutorials or open source projects and would be better to keep them in the original style.